### PR TITLE
add ftp_processor script

### PR DESCRIPTION
**Pull request by VitalySvyatyuk**

### CODE REVIEW

File: ftp_processor.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                        choices='json csv'.split())
+
+
+def main():
+    args = parse_arguments(custom_args)
+
+    ftp = FTP()
+    ftp.connect(args.ftp_host)
+    ftp.login(args.ftp_user, args.ftp_password)
+    ftp.cwd('/drivehqshare/marcemile/GroupWrite')
```
Comment (marcbelmont) [inline ftp_processor.py:65]: this should be configurable

`python ftp_processor.py ... /drivehqshare/marcemile/GroupWrite/ /some/other/path ...`

File: ftp_processor.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    ftp_files = ftp.nlst()
+
+    results = []
+
+    for ftp_file in ftp_files:
+        with tempfile.NamedTemporaryFile(mode='wb') as image:
```
Comment (marcbelmont) [inline ftp_processor.py:71]: Add `logging.info` to display which file is being sent.

File: ftp_processor.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+            ftp.retrbinary(f'RETR {ftp_file}', image.write)
+            if imghdr.what(image.name):  # check is it image or not
+                with open(image.name, 'rb') as fp:
```
Comment (marcbelmont) [inline ftp_processor.py:74]: is that necessary? `image` is already a file pointer

File: ftp_processor.py [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    return args
+
+
+def custom_args(parser):
+    parser.epilog += 'To process images and save result as json: python ftp_processor.py --sdk-url http://localhost:8080  --ftp-host hostname --ftp-user username --ftp-password password\n'
+    parser.add_argument('-t', '--timestamp', help='Timestamp.', required=False)
+    parser.add_argument('--ftp-host', help='FTP host.', required=True)
+    parser.add_argument('--ftp-user', help='FTP user.', required=True)
+    parser.add_argument('--ftp-password', help='FTP password.', required=True)
+    parser.add_argument(
```
Comment (marcbelmont) [inline ftp_processor.py:49]: this argument should have `action='store_true'` instead of requiring a value (-d 1)

File: README.md [hunk 5]
```diff
@@ -101,6 +101,16 @@ To use a locally hosted sdk, pass the url to the docker container as follows:
 
 <br><br><br>
 
+### Process images from the FTP server
```
Comment (marcbelmont) [inline README.md:104]: Update the table of contents at the top of the document too.

File: ftp_processor.py (hunk 4 above)
Comment (VitalySvyatyuk) [reply ftp_processor.py:49]: which one? --ftp-password?

**Pull request merged**